packages/descendants/src/Highlight/README.md

```diff
@@ -0,0 +1,115 @@
+# Highlight Management
+
+A suite of utilities for managing the highlight state of Descendants.
+
+# Usage
+
+```tsx
+/** HighlightDemo.tsx */
+
+const HighlightDemo = () => {
+  // Initialize a new Descendants object
+  const { getDescendants, dispatch } = useInitDescendants<HTMLDivElement>();
+  // Initialize a new highlight state
+  const { highlight, moveHighlight, setHighlight } =
+    useHighlight(getDescendants);
+
+  return (
+    // Init the Descendants context provider
+    <DescendantsProvider
+      context={TestDescendantContext}
+      descendants={getDescendants()}
+      dispatch={dispatch}
+    >
+      {/* Init the Highlight context provider */}
+      <HighlightProvider
+        context={TestHighlightContext}
+        highlight={highlight}
+        setHighlight={setHighlight}
+      >
+        {items.map(item => (
+          <HighlightItem key={item}>{item}</HighlightItem>
+        ))}
+      </HighlightProvider>
+    </DescendantsProvider>
+  );
+};
+```
+
+```tsx
+/** HighlightItem.tsx */
+
+const HighlightItem = ({ children }: PropsWithChildren<{}>) => {
+  const { ref, id } = useDescendant(TestDescendantContext);
+  const { highlight } = useHighlightContext(TestHighlightContext);
```

Comment:
nit: highlight or isHighlighted?

Reply:
it's a `Descendant` object, so `highlight` is correct

Reply:
below we say isHighlighted so just consistent

Reply:
Ah, I accidentally removed the line where I computed `isHighlighted`...